test/passing/comments.ml.opts

```diff
@@ -1 +1 @@
---max-iter=3
\ No newline at end of file
+--max-iter=4
```

Comment:
Does this means that this PR make Ocamlformat a little slower ?

Reply:
That means we need an extra iteration yes (the reason is not obvious from my changes though)

Reply:
fixed